target/linux/rockchip/Makefile

```diff
@@ -0,0 +1,25 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+include $(TOPDIR)/rules.mk
+
+ARCH:=arm
+BOARD:=rockchip
+BOARDNAME:=Rockchip RK33xx
+FEATURES:=ext4 audio usb usbgadget display gpio fpu rootfs-part boot-part
```

Comment:
Please add squashfs for images with f2fs overlay.

Reply:
Done.